SmartDeviceLink-iOS/SmartDeviceLink/SDLInterfaceProtocol.h

```diff
@@ -17,6 +17,7 @@
 - (void)handleBytesFromTransport:(NSData *)receivedData;
 - (void)sendStartSessionWithType:(SDLServiceType)sessionType;
 - (void)sendEndSessionWithType:(SDLServiceType)sessionType sessionID:(Byte)sessionID;
-- (void)sendRPCRequest:(SDLRPCRequest *)rpcRequest;
+- (void)sendRPCRequest:(SDLRPCRequest *)rpcRequest __deprecated_msg("use -sendRPC: instead");
```

Comment:
Because we needed the ability to send RPC Notifications from the app side for foreground / background notifications, which piggyback onHMIStatus.